datagrid_gtk3/db/sqlite.py

```diff
@@ -269,6 +282,14 @@ def update(self, params, ids=None):
                     cursor.execute(sql)
                 conn.commit()
 
+        for (db_file, table_name, id_), db in self.__class__._DBS.items():
+            if db is self:
+                continue
+            if (db_file, table_name) != (self.db_file, self.table.name):
+                continue
+
+            db.emit('rows-changed', params, ids)
```

Comment:
This signal seems to be emitted for all the objects using the same database file and table except the current one. Where is the signal for the current object emitted? A comment in the code to clarify this would be nice to have.